vision_agent/agent/vision_agent_coder.py

```diff
@@ -653,12 +653,10 @@ def __init__(
                  also None, the local python runtime environment will be used.
         """
```

Comment:
Need to change the docstring to AnthropicLMM as we have changed the default planner, coder etc

Reply:
good catch